english/swiper/dora.md

```diff
@@ -0,0 +1,4 @@
+You see two tunnels . Before them is Dora asking you the way to find the marshmellow palace .She asks you which tunnel to take that has no swiper waiting to swipe the marshmellows.oh man!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.